test/VxSortTests/VxSortTests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Do we need to have a dedicated project for VSSort tests? Cannot it be part of SlowTests? Having a new project would require to do the setup on CI

Reply:
It sucks because it is not using our same infrastructure, but I wanted to keep it like the original so I can add more when they add more. I dont even know if our testing frameworks allow to do the generation. Do we have anyone with deep knowledge that can help me to migrate this into our own test framework?

Reply:
You can use `MemberData`: like here https://github.com/ravendb/ravendb/blob/v5.4/test/FastTests/Blittable/BlittableJsonWriterTests/UnmanageJsonReaderTests.cs